requirements.txt

```diff
@@ -12,6 +12,7 @@ linkml-runtime==1.9.4
 hashdir==0.24
 
 # MkDocs
+mkdocs==1.6.1
```

Comment:
@dotdoom do you happen to recall why this `mkdocs==1.6.1` is required?

I can see that it's the same one as currently, which is the latest available.

So far, this somehow got pulled without this... presumably (not checked) as an indirect dependency of `mkdocs-material`.

I'm just curious why it's now required (by Nix?) to explicitly depend on this?

Reply:
I think we don't really need it here.  Removed and pushed, but let's wait for the build.

`mkdocs` was only failing on github actions (not locally) and [only after 20 minutes of build](https://github.com/enola-dev/enola/actions/runs/17217086638/job/48843294266) - very costly to wait! So I went full bananas on it making sure that "it just works" and do cleanups later.

Probably the problem was just nix version overwriting the one from venv.